examples/example.krun

```diff
@@ -75,5 +75,5 @@ TEMP_READ_PAUSE = 1
 #PRE_EXECUTION_CMDS = ["sudo service cron stop"]
 #POST_EXECUTION_CMDS = ["sudo service cron start"]
 
-# CPU pinning (off by default)
-#ENABLE_PINNING = False
+# CPU pinning (on by default)
+#ENABLE_PINNING = True
```

Comment:
We are not mentioning config options in README.md at all. You are right that I do need to update the README a little though.